package io.github.bruno.msavaliadorcredito.exceptions;

public class DataNotFoundException extends Exception{
    public DataNotFoundException(){
        super("Dados n√£o encontrado para o CPF informado ");
    }
}
